build.xml

```diff
@@ -876,7 +882,7 @@ osname=macosx;processor=x86;processor=x86-64;processor=ppc
       <equals arg1="${os.prefix}" arg2="android-armv7"/>
     </condition>
     <condition property="ARCH" value="aarch64">
-      <equals arg1="${os.prefix}" arg2="android-aarch64"/>
+      <matches string="${os.prefix}" pattern="-aarch64$"/>
```

Comment:
Note that this is also duplicated in `native/build.xml`.

Reply:
@fkistner, we couldn't find any documentation or evidence that `native/build.xml` was even used by the project, but I'll patch that up now for consistency.

Reply:
Done. https://github.com/java-native-access/jna/pull/1264/commits/a5e50c6ec1d721ef778e8d6f913cb369c900da9b#diff-28d14c8a63382329901de86493b3db46d5447a0a17f37f240b65687dc203b749R239